fvtest/compilerunittest/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at http://eclipse.org/legal/epl-2.0
+# or the Apache License, Version 2.0 which accompanies this distribution
+# and is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following Secondary
+# Licenses when the conditions for such availability set forth in the
+# Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+# version 2 with the GNU Classpath Exception [1] and GNU General Public
+# License, version 2 with the OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+###############################################################################
+
+cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
+
+project(compunittest LANGUAGES C CXX)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED TRUE)
+set(CMAKE_CXX_EXTENSIONS OFF)
```

Comment:
This may be problematic for other platforms. What C++11 features are we using? Anything "exotic"? We may support some things on z/OS, but not all unfortunately.

Reply:
There's nothing too exotic going on. This is restricted to the same basic C++11 features used in the Tril tests (i.e. `std::tuple` and some very basic variadic templates). Everything in these tests has already been checked to work on AIX when compiled with XLC 13.1, so I doubt there would be any issues on z/OS related to C++11 features that wouldn't also have caused problems there.